helios.js

```diff
@@ -25921,9 +25921,13 @@ export class Address extends CborData {
 	 * Simple payment address without a staking part
 	 * @param {boolean} isTestnet
 	 * @param {PubKeyHash} hash
+	 * @param {?Hash} stakingHash
 	 * @returns {Address}
 	 */
-	static fromPubKeyHash(isTestnet, hash) {
+	static fromPubKeyHash(isTestnet, hash, stakingHash) {
```

Comment:
I would give `stakingHash` a `null` default value so it remains backward compatible.